lambdas/utils/constants/ssm.py

```diff
@@ -0,0 +1,6 @@
+GP_ADMIN_USER_ROLE_CODE = "/auth/smartcard/role/gp_admin_multiple"
```

Comment:
Is it worth naming GP_ADMIN_USER_ROLE_CODES?

Reply:
Potentially, I guess to be clear it's more of an ssm path but GP_ADMIN_USER_ROLE_CODE_SSM_PATH, seemed a bit too long. Can change this if it's not clear